reference/fleet/example-standalone-monitor-nginx-serverless.md

```diff
@@ -1,6 +1,9 @@
 ---
 mapped_pages:
   - https://www.elastic.co/guide/en/fleet/current/example-standalone-monitor-nginx-serverless.html
+applies_to:
+  stack: ga
```

Comment:
This tutorial is specific to Serverless. Step 2: Create an Elastic Cloud Serverless Project. Even if this could work on Stack or Hosted with a few modifications, I don't think we should expect users to fill in those gaps.
Consider removing stack: ga.